README.md

```diff
@@ -198,6 +198,12 @@ export WIREIT_PARALLEL=1
 npm run build
 ```
 
+If two or more seperate `npm run` commands are run for the same Wireit script
+simultaneously, then only one version will be allowed to run at a time, while
```

Comment:
```suggestion
simultaneously, then only one instance will be allowed to run at a time, while
```
?

Talking about "versions" around npm could potentially be confused with package versions.

Reply:
Done